.travis.yml

```diff
@@ -1,61 +1,137 @@
+# Tell Travis to use new container-based infrastructure
 sudo: false
 
+# Declare project language.
+# @link http://about.travis-ci.org/docs/user/languages/php/
 language: php
 
-notifications:
-  email:
-    on_success: never
-    on_failure: change
-
+# Specify when Travis should build.
 branches:
   only:
     - master
 
+# Git clone depth.
+git:
+  depth: 1
+
 matrix:
   include:
-    - php: 5.3
-      env: WP_VERSION=latest
-      dist: precise
     - php: 5.6
-      env: WP_VERSION=latest
+      env: WP_VERSION=latest PHP_LINT=1
     - php: 7.0
-      env: WP_VERSION=latest
+      env: WP_VERSION=latest PHP_LINT=1
     - php: 7.1
-      env: WP_VERSION=latest
+      env: WP_VERSION=latest PHP_LINT=1
     - php: 7.2
-      env: WP_VERSION=latest
-    - php: 5.6
-      env: WP_VERSION=nightly
-    # - php: 5.6
-    #   env: WP_TRAVISCI=phpcs
+      env: WP_VERSION=latest PHP_LINT=1 WP_PHPCS=1
   fast_finish: true
 
+# Use this to prepare the system to install prerequisites or dependencies.
+# e.g. sudo apt-get update.
```

Comment:
You might want to remove this line, even though it's a comment. Long story, but it might affect the platform on which the build is run. Also, it's not very illustrative since you wouldn't want `sudo` to be used. Frankly. you can kill this whole block since `before_install:` is empty anyway.

Reply:
👍